client/my-sites/themes/themes-magic-search-card/style.scss

```diff
@@ -28,7 +28,11 @@
 		flex: 0 1 auto;
 		height: 36px;
 		border: solid 1px var( --color-neutral-10 );
-		margin: 10px;
+		margin: 2rem 10px 10px;
+
+		@include breakpoint-deprecated( '>660px' ) {
```

Comment:
If possible, avoid using the deprecated breakpoints. Here are some other options: p4TIVU-9pb-p2